.github/workflows/pylint.yml

```diff
@@ -23,17 +23,17 @@ jobs:
       run: |
         python -m pip install .
     - name: pycodestyle
-      continue-on-error: true
       run: |
-        python -m pycodestyle --ignore=E501,W503 sed tests 
+        python -m pycodestyle --ignore=E203,E501,W503 sed tests
     - name: pylint
-      continue-on-error: true
+      if: ${{ always() }}
       run: |
-        python -m pylint sed tests 
+        python -m pylint --good-names=i,j,k,ex,x,y,t,k,v,ax,df --disable=fixme sed tests
     - name: mypy
-      continue-on-error: true
```

Comment:
Why is continuing on error a good thing?

Reply:
The old chain was running every item irrespective of its success, but only the last item, the tests would make the entire chain fail.
Now, every item will make the chain fail, but still every item is being executed. If you have neither continue-on-error not if(alyways), the chain would stop on the first failing task, and one would e.g. not run the pytests, if there is a linting error.